src/test/java/us/ihmc/euclid/matrix/RotationMatrixTest.java

```diff
@@ -1106,7 +1101,7 @@ public void testNumericalError() throws Exception
          RotationMatrix unnormalized = new RotationMatrix();
          RotationMatrix normalized = new RotationMatrix();
 
-         for (int i = 0; i < 268000; i++)
+         for (int i = 0; i < 90000; i++)
```

Comment:
Why this number of iterations?

Reply:
Affected directly by the way orientations are now generated: they're "more" random than before. So I'm not surprised that in turn it reduces this number of iterations which represents how fast orientations "denormalize" with operations.